### ISSUE TITLE
Question: Spherical Voronoi Diagram for Generators in a single hemisphere

### ISSUE THREAD

**Issue by fietew** [scipy/scipy#8859]
As pointed out by @tylerjereddy in #5232, the ``scipy.spatial.SphericalVoronoi`` algorithm does not cover cases where all generator points are on a single hemisphere. How would an extension of the algorithm look like to include those cases? Are there theoretical treatises tackling these cases?

### Reproducing code example:
```python
import scipy.spatial
import numpy as np
import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import proj3d                                         
from matplotlib import colors                                      
from mpl_toolkits.mplot3d.art3d import Poly3DCollection

# calculate points in single hemisphere
phi = np.linspace(0,2*np.pi,10,endpoint=False) # azimuth angle
theta = np.linspace(0.001,np.pi*0.4,5)  # polar angle
theta = theta[np.newaxis, :].T
radius = 1;

phiv, thetav = np.meshgrid(phi, theta)
phiv = np.reshape(phiv,(50,1))
thetav = np.reshape(thetav,(50,1))

x = radius*np.cos(phiv)*np.sin(thetav)
y = radius*np.sin(phiv)*np.sin(thetav)
z = radius*np.cos(thetav)

points = np.concatenate([x,y,z],axis=1)

# Voronoi-Regions
sv = scipy.spatial.SphericalVoronoi(points)
sv.sort_vertices_of_regions()

# plotting
fig = plt.figure()                                                              
ax = fig.add_subplot('111', projection='3d')                                    
ax.scatter(points[...,0], points[...,1], points[...,2])    
                                                                                
for region in sv.regions:                                                       
    random_color = colors.rgb2hex(np.random.rand(3))                            
    polygon = Poly3DCollection([sv.vertices[region]], alpha=1.0)                
    polygon.set_color(random_color)                                             
    ax.add_collection3d(polygon)
    
ax.set_xlim([-1.0,1.0])
ax.set_ylim([-1.0,1.0])
ax.set_zlim([-1.0,1.0])
ax.view_init(45,0)

fig.savefig('hemisphere.png', dpi=300) 
```

resulting plot
![hemisphere](https://user-images.githubusercontent.com/7665585/40424201-2e9128a8-5e95-11e8-8ed6-fc13e58c6ded.png)

### Scipy/Numpy/Python version information:
```
1.1.0 1.14.3 sys.version_info(major=3, minor=5, micro=2, releaselevel='final', serial=0)
```

**Comment by fietew** [comment]
> Do we agree on this?

Yes, as near-equatorial generators share the same elevation/polar angle/latitude, their Voronoi regions should share the South Pole as a common Vertex.

**Comment by pmla** [comment]
This issue was resolved by #10494. @tylerjereddy do you want to close?

**Comment by tylerjereddy** [comment]
yep, thanks
